src/style.css

```
@@ -1,3 +1,147 @@
-body {
+html body {
+  font-family: Roboto, Arial, sans-serif;
+  --header-menu-active-color: rgba(0, 172, 220, 1);
+  --menu-apple-width: 37px;
+  --blue-underline-height: 4px;
+  --blue-underline-redius: 30%;
+  --total-length-horizontal-margins-inside-header: 140px;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  background: #fff;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 0;
```

Add properr padding as in Figma designs it is 50px horizontal padding
```suggestion
  padding: 0 50px;
```